File: fuzz/corpus/fuzz_java_parser/seed_simple.java
package com.acme.app;

import java.util.List;

public interface Greeter {
    String greet(String name);
}

class Hello implements Greeter {
    private final String prefix;

    Hello(String prefix) {
        this.prefix = prefix;
    }

    @Override
    public String greet(String name) {
        return prefix + " " + name;
    }
}

File: fuzz/corpus/fuzz_kotlin_parser/seed_simple.kt
package com.acme.app

import kotlin.math.max

interface Greeter {
    fun greet(name: String): String
}

data class Hello(val prefix: String) : Greeter {
    override fun greet(name: String): String = "$prefix $name"
}

fun main() {
    println(Hello("hello").greet("world"))
}
